src/components/FormSelectInput.component.tsx

```diff
@@ -0,0 +1,70 @@
+import React from "react";
+import styled from "styled-components";
+
+import { defaultTheme } from "../theme";
+import { InputStyle } from "./styled";
+
+type MapElement = { id: number; name: string };
+interface State extends MapElement {
+  cities: Array<MapElement>;
+}
+export const FormSelectInput = ({
+  label,
+  register,
+  onFocus,
+  onBlur,
+  onChange,
+  loading,
+  data,
+  disabled,
+}: {
+  label: string;
+  register: any;
+  onFocus?: any;
+  onBlur?: any;
+  onChange?: (e: React.ChangeEvent) => void;
+  loading: boolean;
+  data: any;
+  disabled?: boolean;
+}) => {
+  return (
+    <Select
+      {...register}
+      onFocus={onFocus}
+      onChange={onChange ? onChange : register.onChange}
+      disabled={disabled}
+      defaultValue=""
```

Comment:
what if someone using this component wants to set a default value, how do you think you can make this component more flexible for these scenarios?

Reply:
In that case we could add that parameter as a prop and set it inside the component, am I right?